app/schemas.py

```diff
@@ -125,36 +125,37 @@ def to_orm(self, provider):
 
     class Config:
         orm_mode = True
+        from_attributes = True
 
 
 @dataclass
 class QueryParams:
-    source: str = Query(
+    source: Optional[str] = Query(
```

Comment:
curious, why these explicit "optional" are needed?

Reply:
Because some tests send None in the source field?